packages/jackett/build.sh

```diff
@@ -0,0 +1,44 @@
+TERMUX_PKG_HOMEPAGE="https://github.com/jackett/jackett"
+TERMUX_PKG_DESCRIPTION="API Support for your favorite torrent trackers"
+TERMUX_PKG_LICENSE="GPL-3.0"
+TERMUX_PKG_MAINTAINER="@termux"
+TERMUX_PKG_VERSION="0.22.1428"
+TERMUX_PKG_SRCURL="https://github.com/Jackett/Jackett/archive/refs/tags/v${TERMUX_PKG_VERSION}.tar.gz"
+TERMUX_PKG_SHA256="71caabacc957e0b98e3f425b76099000ac80dacf22077ffcba5f012fb1bb12f1"
+TERMUX_PKG_BUILD_DEPENDS="aspnetcore-targeting-pack-8.0, dotnet-targeting-pack-8.0"
+TERMUX_PKG_DEPENDS="aspnetcore-runtime-8.0, dotnet-host, dotnet-runtime-8.0"
+TERMUX_PKG_BUILD_IN_SRC=true
+TERMUX_PKG_SERVICE_SCRIPT=("jackett" 'exec ${TERMUX_PREFIX}/bin/jackett --DataFolder ${TERMUX_ANDROID_HOME}/.config/jackett 2>&1')
+TERMUX_PKG_BLACKLISTED_ARCHES="arm"
+TERMUX_PKG_RM_AFTER_INSTALL="
+lib/jackett/libMono.Unix.so
+"
+
+termux_step_pre_configure() {
+	export DOTNET_CLI_TELEMETRY_OPTOUT=1
+	termux_setup_dotnet
+}
+
+termux_step_make() {
+	dotnet publish src/Jackett.Server \
+	--framework "net8.0" \
+	--no-self-contained \
+	--runtime "$DOTNET_TARGET_NAME" \
+	--configuration Release \
+	--output build/ \
+	/p:AssemblyVersion="${TERMUX_PKG_VERSION}" \
+	/p:FileVersion="${TERMUX_PKG_VERSION}" \
+	/p:InformationalVersion="${TERMUX_PKG_VERSION}" \
+	/p:Version="${TERMUX_PKG_VERSION}"
+	dotnet build-server shutdown
+}
+
+termux_step_make_install() {
+	cp -r build "${TERMUX_PREFIX}/lib/jackett"
+	cat > $TERMUX_PREFIX/bin/jackett <<HERE
+#!$TERMUX_PREFIX/bin/sh
+export LD_LIBRARY_PATH=$TERMUX_PREFIX/lib:\$LD_LIBRARY_PATH
```

Comment:
This should not be necessary. Please set rpath instead, by passing `-Wl,-rpath=$TERMUX_PREFIX/lib` in some linker flag.

Reply:
I don't know anything about dotnet and what compiler flags it uses, so cannot give a full command. For many systems it is enough to:

```
export LDFLAGS+=" -Wl,-rpath=$TERMUX_PREFIX/lib"
```

but maybe dotnet does things differently.

Reply:
I don't want jackett to rely on LD_LIBRARY_PATH in any case, please find a way to set rpath instead.

Reply:
I already specify if use dotnet to launch dll files, you won't need to specify LD_LIBRARY_PATH since that responsibility goes to dotnet to handle (MY responsibility as the dotnet package maintainer and is on my TODO list). If you use the jackett binary, the responsibility lies on you.